### DISCUSSION TITLE
Weather warnings

### DISCUSSION THREAD

**Discussion by Vali6053** [siku2/hass-srf-weather#15] [Ideas]
Is it also possible to bring the weather warnings from srf meto to the home assistant?  So that the blinds go up when a message comes up?

**Comment by siku2** [comment]
Just browsed through the API documentation and it doesn't look like this information is exposed, unfortunately.
Can you tell me where you usually get these alerts from, @Vali6053?

**Comment by Vali6053** [comment]
I get the notifications from the srf meteo app on my cell phone, thought it might be possible to transfer the thunderstorm and storm warnings to the home assistant.

**Comment by siku2** [comment]
Yeah, I would love to have that as well. We just need to find a source to grab it from.

**Comment by Vali6053** [comment]
The only thing I found is this website, you have to register officially so that you get an api interface.
https://www.hagelschutz-einfach-automatisch.ch/eigentuemer-verwaltungen/produkt/anmeldung-rest-api.html
